### DISCUSSION TITLE
MAX6675 Temperature is not working on ESP8266 but on ESP32S2

### DISCUSSION THREAD

**Discussion by canis-ferrin** [arendst/Tasmota#21906] [Q&A]
Seems to be a problem with platformio.
I already reported such a bug with a servo.

**Comment by Jason2866** [comment]
There is no relevant info in your post. What exactly is not working in Tasmota?

**Comment by canis-ferrin** [comment]
MAX6675 is not recognized from Wemos D1 mini when firmware is compiled in tasmota-4M mode and send to the Wemos D1 mini. 
Web interface shows a value of 0 °C for the temperature. SetOption94 1 is set.
When compiled the same firmware configuration in tasmota32s2 mode and send to a ESP32 S2 Mini V1.0.0,
it shows MAX6675 Temperature 24.50 °C, which is a true value. SetOption94 1 is set.
Btw: Same issue is with shutter and blinds using an A4988 driven stepper motor.
